ledger-service/http-json/src/main/scala/com/digitalasset/http/EndpointsCompanion.scala

```diff
@@ -195,99 +172,52 @@ object EndpointsCompanion {
 
   }
 
-  object ParsePayload {
-    @inline def apply[A](implicit ev: ParsePayload[A]): ParsePayload[A] = ev
+  object CreateFromCustomToken {
+    @inline def apply[A](implicit ev: CreateFromCustomToken[A]): CreateFromCustomToken[A] = ev
```

Comment:
I don't get the need for this code? why have an apply with an implicit, to then just return the implicitly passed argument?

Reply:
I don't understand it either but given that this is existing code I would not block this PR on addressing it. Maybe create a gardening ticket for investigation & possible removal?

Reply:
It's effectively a convenience method for `implicitly`; defining this method is popular when users of your instance prefer to use the `A: TC` context bound on methods instead of writing out a whole `implicit A: TC[A]` argument (and argument list), _and_ wish to refer to the instance in the method body, like so: `TC[A]` instead of `implicitly[TC[A]]`. So when you want to get "the `TC[A]` instance" you literally write `TC[A]`.

I tend to prefer the `implicit A: TC[A]` er, "explicit" argument myself, which obviates the desire for this method, but I admit that it pollutes the local namespace a bit.